.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "substrate"]
 	path = substrate
-	url = https://github.com/paritytech/substrate.git
+	url = https://github.com/Phala-Network/substrate.git
```

Comment:
We have a plan to switch the cargo dependency of substrate to the original one on git. It's necessary to consolidate  phala-blockchain and khala-parachain. However from this PR, we have made some change on substrate. So we need to figure out what's the best way...

Reply:
Is there a reason that require us to switch to `cargo git dep`?
I think submodule is easier to maintain.

Reply:
In the future we are going merge khala-parachain back to this repo. As a result, we are going to rely on `substrate`, `cumulu`, and `polkadot` all together. When we only rely on Substrate it was not a big deal because it's mostly self contained. But once we introduce `cumulus` and `polkadot`, they rely on each other as git crates. If we still put them as a submodule, we will have to fork all of them and always constantly change their reference in Cargo.toml. It will be painful.

Reply:
Switched back to unmodified substrate.